packages/dev-utils/src/lib/geo-location.ts

```diff
@@ -0,0 +1,12 @@
+import type { Context } from '@netlify/types'
+
+export type Geolocation = Context['geo']
+
+export const mockLocation: Geolocation = {
```

Comment:
We should incorporate the CLI logic that uses https://netlifind.netlify.app/ to provide a real geo-location object.